packages/ui/src/components/search-box.tsx

```diff
@@ -78,11 +80,9 @@ const Root = forwardRef<HTMLInputElement, SearchBoxProps>(
     }
 
     const handleFocus = () => {
-      if (onFocus) {
-        onFocus()
-      }
+      onFocus?.()
       if (showOnFocus) {
-        handleSearch()
+        handleSearch?.()
```

Comment:
`handleSearch` will always exist since it's defined on line 71
```suggestion
        handleSearch()
```